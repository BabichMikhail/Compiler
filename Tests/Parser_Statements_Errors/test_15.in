begin 
	a := 100
	b := 50	
end.