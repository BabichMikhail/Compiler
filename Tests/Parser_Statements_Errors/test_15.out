Fatal: Syntax error, ";" expected but "b" found in Line 3 Column 5
